packages/teams-js/test/public/tasks.spec.ts

```diff
@@ -126,7 +126,12 @@ describe('tasks', () => {
   });
 
   describe('updateTask', () => {
-    const allowedContexts = [FrameContexts.task];
+    const allowedContexts = [
```

Comment:
Personally I'm still unsure whether these changes should go in dialog vs. tasks vs. both. I'm leaning towards just dialog since tasks.updateTask is deprecated and what's really changing underneath the hood is in dialog, and having it in both seems duplicated. If you have a strong-ish opinion for any of these options, let me know! If not, I think we could move it to dialog and make a ticket to have someone else double-check what the situation here and other similar circumstances should generally point to. (i.e. changes in deprecated functions from v1 should be tested in the new function's test file vs. the deprecated function's test file)

Reply:
If we don't propagate this here, the tests will fail due to tasks calling `dialog.open` under the hood. Sadly I don't think we can remove it just yet, but we can definitely make a ticket to remove it in the future!